internal/pkg/flink/internal/store/store.go

```diff
@@ -104,39 +104,29 @@ func (s *Store) FetchStatementResults(statement types.ProcessedStatement) (*type
 	}
 
 	// Process remote statements that are now running or completed
-	pageToken := statement.PageToken
-	runningNoTokenRetries := 5
-	for i := 0; i < runningNoTokenRetries; i++ {
-		// TODO: we need to retry a few times on transient errors
-		statementResultObj, err := s.client.GetStatementResults(s.appOptions.GetOrgResourceId(), s.appOptions.GetEnvId(), statement.StatementName, pageToken)
-		if err != nil {
-			return nil, &types.StatementError{Msg: err.Error()}
-		}
+	statementResultObj, err := s.client.GetStatementResults(s.appOptions.GetOrgResourceId(), s.appOptions.GetEnvironmentId(), statement.StatementName, statement.PageToken)
+	if err != nil {
+		return nil, &types.StatementError{Msg: err.Error()}
+	}
 
-		statementResults := statementResultObj.GetResults()
-		convertedResults, err := results.ConvertToInternalResults(statementResults.GetData(), statement.ResultSchema)
-		if err != nil {
-			return nil, &types.StatementError{Msg: "Error: " + err.Error()}
-		}
-		statement.StatementResults = convertedResults
+	statementResults := statementResultObj.GetResults()
+	convertedResults, err := results.ConvertToInternalResults(statementResults.GetData(), statement.ResultSchema)
+	if err != nil {
+		return nil, &types.StatementError{Msg: fmt.Sprintf("Error: %v", err.Error())}
+	}
+	statement.StatementResults = convertedResults
 
-		statementMetadata := statementResultObj.GetMetadata()
-		extractedToken, err := extractPageToken(statementMetadata.GetNext())
-		if err != nil {
-			return nil, &types.StatementError{Msg: "Error: " + err.Error()}
-		}
-		statement.PageToken = extractedToken
-		if statement.Status == types.COMPLETED || statement.PageToken != "" || len(statementResults.GetData()) > 0 {
-			// We try a few times to get non-empty token for RUNNING statements
-			break
-		}
-		time.Sleep(time.Millisecond * 300)
+	statementMetadata := statementResultObj.GetMetadata()
+	extractedToken, err := extractPageToken(statementMetadata.GetNext())
+	if err != nil {
+		return nil, &types.StatementError{Msg: fmt.Sprintf("Error: %v", err.Error())}
```

Comment:
```suggestion
		return nil, &types.StatementError{Msg: fmt.Sprintf("error: %v", err)}
```

same as above

Reply:
It seems to be the convention in this repo (at least from what I have seen so far). Every message that gets shown to the user/printed to the terminal always starts with an uppercase letter